src/main/java/org/bytedeco/javacpp/Pointer.java

```diff
@@ -124,6 +125,13 @@ void init(long allocatedAddress, long allocatedCapacity, long ownerAddress, long
         limit = allocatedCapacity;
         capacity = allocatedCapacity;
         if (ownerAddress != 0 && deallocatorAddress != 0) {
+            if (nativeAllocationTracerEnabled) {
+                try {
+                    tracerMarkPointerMethod.invoke(null, this);
+                } catch (IllegalAccessException | InvocationTargetException exception) {
```

Comment:
Let's just catch ReflectiveOperationException for simplicity

Reply:
Oh, thanks. Applied.